### ISSUE TITLE
In the social feed, tags don’t link to a tag-based search (which we DO have for content)

### ISSUE THREAD

**Issue by bfackland** [ploneintranet/ploneintranet#855]
We don't yet have social search, but I don't think that should stop us hyperlinking tags in the social feed to a search for "all content with that tag".

![screen shot 2015-10-22 at 12 18 35](https://cloud.githubusercontent.com/assets/2538526/10663945/42fc6b84-78b7-11e5-8d08-216cf0e33ffa.png)

![screen shot 2015-10-22 at 12 19 27](https://cloud.githubusercontent.com/assets/2538526/10663944/42d70088-78b7-11e5-969e-7d8d895726ae.png)

**Comment by mattss** [comment]
I can't at the moment find an examples in the prototype for where social tags should link to.  I'm slightly concerned that making links from social posts to a site-wide content search (that doesn't yet include social posts) would be confusing for a user.

@gyst @cornae can you point me to anything in the prototype to indicate how social hashtags should operate?

**Comment by gyst** [comment]
No we don't have a design, nor do we have a budget to work on that, which is why the developer-centric "implementation" of this was recently removed.
https://github.com/ploneintranet/ploneintranet/issues/689

I.e. tag search on microblog posts is already supported in the backend, we're just lacking the actual frontend design. In any case it should be separate from 'content' search until the day we have full-text indexing of all microblog posts.

**Comment by bfackland** [comment]
We will also need to consider where links to status updates actually go - e.g. you search for a term, see previews of results and then want to view the full conversation in context. At present there is no way to link to view a single status update with follow-up comments.

**Comment by cornae** [comment]
I think status updates should not be shown in search results until there’s a design for this page. Which shouldn’t be a large effort by the way.

**Comment by gyst** [comment]
This is now confusing 3 different things:
1. We do have an implementation for tag-searching the microblog. The results page for that has not been designed yet. This is not a search page but a stream view.
2. We do have a page for showing only one conversation (status update). This is a pre-requirement for 3.
3. We do not yet index microblog statusupdates in such a way that they can be searched from Solr.

So the key question is, do we aim for (1) styling a stream view variant for a tag (backend already available) or do we aim for (3) solr search of statusupdates.

We need (3) in the long term. The question is sorting: should it be sorted reverse chronologically (stream) or by text relevance (default solr).

**Comment by gyst** [comment]
Actually we don't have (2) yet, see #486

**Comment by bfackland** [comment]
I agree. Do you have an idea for how much work would be involved in prototyping the status updates-search integration?

> On 3 Nov 2015, at 12:30, Cornelis G. A. Kolbach notifications@github.com wrote:
> 
> I think status updates should not be shown in search results until there’s a design for this page. Which shouldn’t be a large effort by the way. 
> 
> > Op 3 nov. 2015, om 11:44 heeft Ben Ackland notifications@github.com het volgende geschreven:
> > 
> > We will also need to consider where links to status updates actually go - e.g. you search for a term, see previews of results and then want to view the full conversation in context. At present there is no way to link to view a single status update with follow-up comments.
> > 
> > —
> > Reply to this email directly or view it on GitHub.
> 
> —
> Reply to this email directly or view it on GitHub.

## 

Ben Ackland, Projects Director
Netsight Internet Solutions Limited
Tel: +44 (0)117 90 90 90 1

Registered in England no. 3892180
Registered office: 40 Berkeley Square, Bristol, BS8 1HU, United Kingdom

**Comment by cornae** [comment]
A page for a single status page with discussion 1 to 3 hours. A search result for a post we already have, albeit not yet with multi page previews or pages with multiple attachments like we have currently on the stream in the prototype. That would add another 1 to 3 hours.
